config/samples/gateway-with-gatewayconfiguration.yaml

```diff
@@ -18,9 +18,9 @@ spec:
           - name: proxy
             # renovate: datasource=docker versioning=docker
             image: kong/kong-gateway:3.9
-            readinessProbe:
-              initialDelaySeconds: 1
-              periodSeconds: 1
+            # readinessProbe:
+            #   initialDelaySeconds: 7
+            #   periodSeconds: 8
```

Comment:
?

Reply:
Accidental change. 460da16f3dfcfe07548a84c7d97a503eb322b4c6 fixes it. PTAL.